calcom/cal.com
# Save meeting url to metadata of bookings and add it to webhook payload

## What does this PR do?

Adds new metadata column to booking model to save the meeting URL. Also adds the meeting URL to webhook payload.

Webhook payload for booking created and rescheduled: 

With Cal Video link: 
![Screenshot 2022-11-29 at 14 54 32](https://user-images.githubusercontent.com/30310907/204547996-d3f835cc-aa87-41c3-9e1e-8747fb9d5bd9.png)

With Google Meet link: 
![Screenshot 2022-11-29 at 14 55 23](https://user-images.githubusercontent.com/30310907/204548044-0e5515bc-3a7a-4d9b-8894-c42c1402242e.png)

Fixes #5518

**Environment**: Staging(main branch)

## How should this be tested?

- Book an event type with a video call app as a location (Cal Video, Google Meet, Zoom, ...)
- Check database if videoCallUrl is saved in metadata 
- Created a webhook and test if it returns the video call data when it is triggered
